build.gradle

```diff
@@ -35,70 +38,51 @@ ext.deps = [
 ]
 
 dependencies {
+
     compile 'com.android.support:appcompat-v7:' + deps.androidSupport
     compile 'com.android.support:support-annotations:' + deps.androidSupport
     compile 'com.android.support:design:' + deps.androidSupport
-
     compile 'com.google.android.gms:play-services-analytics:' + deps.playServices
     compile 'com.google.android.gms:play-services-gcm:' + deps.playServices
-
     compile 'org.ccci.gto.android:gto-support-core:' + deps.gtoSupport
     compile 'org.ccci.gto.android:gto-support-db:' + deps.gtoSupport
     compile 'org.ccci.gto.android:gto-support-db-async:' + deps.gtoSupport
     compile 'org.ccci.gto.android:gto-support-gson:' + deps.gtoSupport
-
     compile 'com.crashlytics.sdk.android:crashlytics:2.6.5'
     compile 'com.newrelic.agent.android:android-agent:5.8.4'
-
     compile 'com.google.guava:guava:' + deps.guava
-    compile 'com.squareup.picasso:picasso:2.5.2'
     compile 'com.jakewharton:butterknife:' + deps.butterknife
     compile 'com.squareup.retrofit2:retrofit:' + deps.retrofit2
     compile 'com.squareup.retrofit2:converter-gson:' + deps.retrofit2
-
     compile 'org.apache.commons:commons-lang3:3.4'
-
-    compile 'org.greenrobot:eventbus:3.0.0'
-
-    debugCompile 'com.facebook.stetho:stetho:' + deps.facebookStetho
-    debugCompile 'com.facebook.stetho:stetho-okhttp3:' + deps.facebookStetho
```

Comment:
I would love to be able to keep stetho on debug builds for development usage. I know it was pulled in order to avoid multi-dex.